site/_docs/configuration.md

```diff
@@ -571,11 +571,11 @@ file or on the command-line.
 # Where things are
 source:       .
 destination:  ./_site
-plugins_dir:  ./_plugins
-layouts_dir:  ./_layouts
-data_dir:     ./_data
-includes_dir: ./_includes
-collections:  null
+plugins_dir:  _plugins
+layouts_dir:  _layouts
+data_dir:     _data
+includes_dir: _includes
+collections:  {}
```

Comment:
It doesn't [appear](https://github.com/jekyll/jekyll/blob/master/lib/jekyll/configuration.rb#L15) to be.